### PR TITLE
feat: Add /map endpoint to serve optimized Place data for Map Integration

### PR DESCRIPTION
**Pull request by cyaiox**
This PR introduces a new `/map` endpoint that provides a streamlined dictionary of Place data, keyed by each Place's `base_position`. The endpoint returns a reduced set of Place information optimized for map fetching by the Explorer, enabling efficient data retrieval and integration.

Returned object:
```
{
  "total": 1,
  "ok": true,
  "data": {
    "1,1": {
      "id": "xyz-xyz-xyz",
      "base_position": "1,1",
      "title": "Test Title",
      "description": "Test Description",
      "image": "https://peer.decentraland.org/.../someImg",
      "contact_name": "Test",
      "categories": [
        "art",
        "poi"
      ],
      "user_favorite": false,
      "user_like": false,
      "user_dislike": false,
      "user_visits": 100,
      "user_count": 50
    },
  }
}
```

Closes: https://github.com/decentraland/places/issues/551

### CODE REVIEW

File: src/entities/Map/routes/getMapPlacesMostActive.ts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      order:
+        oneOf(ctx.url.searchParams.get("order"), ["asc", "desc"]) || "desc",
+      with_realms_detail: ctx.url.searchParams.get("with_realms_detail"),
+      search: ctx.url.searchParams.get("search"),
+      categories: ctx.url.searchParams.getAll("categories"),
+    })
+
+    const sceneStats = await getSceneStats()
+
+    const hotScenes = getHotScenes()
```
Comment (braianj) [inline src/entities/Map/routes/getMapPlacesMostActive.ts:39]: This is a mix of environments. getHotScenes was defined in modules, that's part of the frontend, but entities are part of the backend. Would you mind adding a similar function inside utils?

File: src/entities/Map/routes/getMapPlacesMostActive.ts [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      only_highlighted: ctx.url.searchParams.get("only_highlighted"),
+      order_by: PlaceListOrderBy.MOST_ACTIVE,
+      order:
+        oneOf(ctx.url.searchParams.get("order"), ["asc", "desc"]) || "desc",
+      with_realms_detail: ctx.url.searchParams.get("with_realms_detail"),
+      search: ctx.url.searchParams.get("search"),
+      categories: ctx.url.searchParams.getAll("categories"),
+    })
+
+    const sceneStats = await getSceneStats()
```
Comment (braianj) [inline src/entities/Map/routes/getMapPlacesMostActive.ts:37]: Same as with getHotScenes

File: src/entities/Map/utils.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    let stats: SceneStats | undefined = sceneStats[place.base_position]
+
+    if (!stats) {
+      const statsPosition = (place.positions || []).find(
+        (position) => sceneStats[position]
+      )
+      if (statsPosition) {
+        stats = sceneStats[statsPosition]
+      }
+    }
```
Comment (braianj) [inline src/entities/Map/utils.ts:28]: Would you mind avoiding [nest](https://www.youtube.com/watch?v=CFRhGnuXG-4) code? 
Maybe something like
```suggestion
    let statsPosition
    if (!stats) {
      statsPosition = (place.positions || []).find(
        (position) => sceneStats[position]
      )
    }
    if (!stats && statsPosition) {
        stats = sceneStats[statsPosition]
    }
```

**Review by braianj — CHANGES_REQUESTED** [review]
Awesome work 🎉 

Here are some short comments

File: src/entities/Map/routes/getMapPlacesMostActive.ts (hunk 2 above)
Comment (cyaiox) [reply src/entities/Map/routes/getMapPlacesMostActive.ts:37]: updated!

File: src/entities/Map/routes/getMapPlacesMostActive.ts (hunk 1 above)
Comment (cyaiox) [reply src/entities/Map/routes/getMapPlacesMostActive.ts:39]: updated!

**Review by braianj — APPROVED** [review]
Great work! 🎉

**Pull request merged**